lib/config.js

```diff
@@ -56,12 +56,27 @@ module.exports = {
         containerName: config.get('COMPONENT_AZBLOB_CONTAINER_NAME') || config.get('HARVEST_AZBLOB_CONTAINER_NAME') + '-component' || `component-${process.env.NODE_ENV}`
       },
       file: {
-        location: config.get('FILE_STORE_LOCATION') || process.platform === 'win32' ? 'c:/temp/cd' : '/tmp/cd'
+        location: config.get('FILE_STORE_LOCATION') || (process.platform === 'win32' ? 'c:/temp/cd' : '/tmp/cd')
       }
     }
   },
   auth: {
     apiToken: config.get('API_TOKEN'),
-    password: config.get('AUTH_SITE_PASSWORD')
+    password: config.get('AUTH_SITE_PASSWORD'),
```

Comment:
do we still need this config option? Seems like it was only used in the old, basic auth support.

Reply:
Probably not. Removing.